src/widgets/responsive_images.js

```diff
@@ -52,8 +53,24 @@ filepicker.extend('responsiveImages', function(){
         removeWindowResizeEvent(reloadWithDebounce);
     }
 
+    function update(element){
+        if (element !== undefined) {
+            // If <img> given as a parameter then just process it directly.
+            if (element.nodeName === 'IMG') {
+                construct(element);
+            } else {
+                throw new Error('Passed object is not an image');
+            }
+        } else {
+            var responsiveImages = document.querySelectorAll('img[data-fp-src]');
+            for (var i = 0; i < responsiveImages.length; i++) {
+                construct(responsiveImages[i]);
+            }
```

Comment:
I saw this when I pulled the repo, but I do not know the correct way to call it.... I made my own hack to make it work... but I would rather use the official release.